layers/+emacs/outshine/packages.el

```diff
@@ -0,0 +1,91 @@
+;;; packages.el --- outshine layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2012-2019 Sylvain Benner & Contributors
+;;
+;; Author: Langston Barrett <langston.barrett@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+;;; Commentary:
+
+;; Briefly, each package to be installed or configured by this layer should be
+;; added to `outshine-packages'. Then, for each package PACKAGE:
+;;
+;; - If PACKAGE is not referenced by any other Spacemacs layer, define a
+;;   function `outshine/init-PACKAGE' to load and initialize the package.
+
+;; - Otherwise, PACKAGE is already referenced by another Spacemacs layer, so
+;;   define the functions `outshine/pre-init-PACKAGE' and/or
+;;   `outshine/post-init-PACKAGE' to customize the package as it is loaded.
+
+;;; Code:
+
+(defconst outshine-packages
+  '(outshine
+    outorg))
+
+(defun outshine/init-outshine ()
+  (use-package outshine
+    :defer t
+    :init
+    (progn
+      (add-hook 'prog-mode-hook          'outline-minor-mode)
+      (add-hook 'outline-minor-mode-hook 'outshine-mode))
+    :config
+    (progn
+       (spacemacs/declare-prefix "aO" "outline")
+       (spacemacs/set-leader-keys
+         "aO."  'spacemacs/outshine-transient-state/body
+         "aOS"  'outline-show-all
+         "aOgu" 'outline-up-heading
+         "aOgn" 'outline-next-heading
+         "aOgj" 'outline-forward-same-level
+         "aOgk" 'outline-backward-same-level
+         "aOih" 'outline-insert-heading
+         "aOI"  'outshine-imenu
+         "aOK"  'outline-move-subtree-up
+         "aOJ"  'outline-move-subtree-down
+         "aO>"  'outline-demote
+         "aO<"  'outline-promote))
+       (spacemacs|define-transient-state outshine
+         :title "Outshine Transient State"
+         :doc "
+     Navigate headings^^^^      Move subtrees^^                 Other^^
+     ─────────────────^^^^──    ─────────────^^───────────────  ─────^^───
+     [_j_/_k_] down/up          [_J_/_K_] move subtree down/up  [_q_] quit
+     [_n_/_N_] next/up heading  [_>_/_<_] demote/promote        [_i_] insert heading
+     [_I_] heading imenu"
```

Comment:
```suggestion
Navigate headings^^^^      Move subtrees^^^^               Other^^
─────────────────^^^^────  ─────────────^^^^─────────────  ─────^^─────────────
[_j_/_k_] down/up          [_J_/_K_] move subtree down/up  [_q_] quit
[_n_/_N_] next/up heading  [_>_/_<_] demote/promote        [_i_] insert heading
[_I_]^^   heading imenu"
```
Fixed the Transient state alignment.
The `^` (caret) characters have to match the number of underscores in each column.